specification/confidentialledger/Microsoft.CodeTransparency/main.tsp

```diff
@@ -28,10 +27,764 @@ namespace Microsoft.CodeTransparency;
+@route("/.well-known/did.json")
+@get
+op getDidConfig is Foundations.Operation<{}, DidDocument, ServiceTraits>;
+
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+@doc("Get CodeTransparency configuration information.")
+@route("/configuration")
+@get
+op getCodeTransparencyConfig is Foundations.Operation<
+  {},
+  CodeTransparencyConfiguration,
+  ServiceTraits,
+  JsonServerErrors
+>;
+
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+@doc("Get CodeTransparency version information.")
+@route("/version")
+@get
+op getCodeTransparencyVersion is Foundations.Operation<
+  {},
+  VersionResult,
+  ServiceTraits,
+  JsonServerErrors
+>;
+
+// -----------------------------------------------------------------
+// Service entry endpoints
+// -----------------------------------------------------------------
+
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+@removed(Versions.`2025-01-31-preview`)
+@doc("Post an entry to be registered on the CodeTransparency instance (deprecated)")
+@post
+@sharedRoute
+@route("/entries")
+op createEntryDeprecated is Foundations.Operation<
+  SignedStatement,
+  CreateEntryResult,
+  ServiceTraits
+>;
+
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+@added(Versions.`2025-01-31-preview`)
+@doc("Post an entry to be registered on the CodeTransparency instance, see IETF SCITT draft")
+@post
+@sharedRoute
+@route("/entries")
+op createEntry is Foundations.Operation<
+  SignedStatement,
+  CreateEntryCreated | CreateEntryPending,
+  ServiceTraits,
+  AnyCborError
+>;
+
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+@removed(Versions.`2025-01-31-preview`)
+@doc("Get the status the entry operation (deprecated)")
+@route("/operations/{operationId}")
+op getEntryStatus is Foundations.Operation<
+  OperationIdParameter,
+  GetOperationResult,
+  ServiceTraits
+>;
+
+#suppress "@azure-tools/typespec-azure-core/use-standard-names" "Existing name"
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+@removed(Versions.`2025-01-31-preview`)
+@doc("Get the status of all entries operations (deprecated)")
+@route("/operations")
+op listEntryStatuses is Foundations.Operation<
+  {},
+  ListOperationResult,
+  ServiceTraits
+>;
+
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+@removed(Versions.`2025-01-31-preview`)
+@doc("Get signed statement with or without receipt embedded in its unprotected header")
+@sharedRoute
+@route("/entries/{entryId}")
+op getEntryDeprecated is Foundations.Operation<
+  EmbedReceiptParameter,
+  SignedStatement | TransparentStatement,
+  ServiceTraits
+>;
+
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+@added(Versions.`2025-01-31-preview`)
+@doc("Get signed statement with or without receipt embedded in its unprotected header")
+@sharedRoute
+@route("/entries/{entryId}")
+op getEntry is Foundations.Operation<
```

Comment:
as tested during SDK generation, this should be 

```
op getEntry is Foundations.Operation<
  {
    @doc("Accept header")
    @header
    accept: "application/cose";
  } &
  EmbedReceiptParameter,
  SignedStatement | TransparentStatement,
  ServiceTraits,
  AnyCborError
>;

```

Reply:
Thanks, updated